### PR TITLE
Add styling, font, and extra resources

### PR DESCRIPTION
**Pull request by jnnybk**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,39 +1,65 @@
...
-			</table>
-
-				
-			<div id="main-section">
-				<img id="headshot" src="/images/Headshot_Jenny.JPG" alt="Image of me."/>
-				<p>Click here to see a random fact about me!:</p>
-				<button onclick="addRandomGreeting()">Hello</button>
-				<div id="greeting-container"></div>
+
+<head>
```
Comment (belraquib) [inline portfolio/src/main/webapp/index.html:4]: The indentation seems weird. Can you run the formatter again?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,39 +1,65 @@
...
+				<br/>
+				<h3>Some of her favorite moments!</h3>
+					<ul>
+						<li>Learning how to make Japanese cheesecake without the sour cream mixture separating</li>
+						<li>Thanksgiving of 2019 - Friendsgiving (Food) Party where she made BBQ chicken sliders from scratch</li>
+						<li>Meeting fellow CS interns over community chats</li>
+					</ul>
+				<h3>Songs that she's currently listening to!</h3>
+					<table>
+						<th><a style="color: #4a4e69;text-decoration:none;" href="https://youtu.be/rEGOihjqO9w">Rach's 2nd</a></th>
```
Comment (belraquib) [inline portfolio/src/main/webapp/index.html:55]: one of my favorites!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (jnnybk) [reply portfolio/src/main/webapp/index.html:4]: Hm, I'm not sure why it shows up on GitHub like this. In the CloudShell editor, the code seems to be correctly indented (using two spaces). I tried fixing it, and ran git diff, but there is no difference.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,39 +1,65 @@
...
 			</div>
 
 			<div id="contact-section">
-				<button link="https://www.instagram.com/waikandbaik/">Visit my food instagram!</button>
-				<button link="https://www.linkedin.com/jennybaik206">Connect with me on LinkedIn!</button>
-				<button link="mailto:baikj@google.com">Email me!</button>
+				<img src="/images/glyph-logo_May2016.png" usemap="#instagram"/>
+				<map name="instagram">
+					<area shape="rect" coords="0,0,30,30" href="https://www.instagram.com/waikandbaik/"/>
+				</map>
```
Comment (mickrobk) [inline portfolio/src/main/webapp/index.html:30]: Why use a shape map here? (If you're making the whole image a link, you can alternatively wrap the  in an <a href>)

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,24 +1,90 @@
...
 }
 #contact-section {
-  display: none
+  display: none;
+}
+#about-section {
+	display: none;
+}
+body {
+	color: rgb(34, 34, 59);
```
Comment (belraquib) [inline portfolio/src/main/webapp/style.css:24]: the spacing is inconsistent, can you reformat?

**Pull request merged**